### ISSUE TITLE
Refactor Rename does not respect whether a class actually implements an interface

### ISSUE THREAD

**Issue by MDoerner** [rubberduck-vba/Rubberduck#3061]
Refactor rename will rename members of other classes matching the naming convention regardless whether the interface is actually implemented.

Interface:
```vba
Public Sub Foo()
End Sub
```
Class:
```vba
Public Sub Interface_Foo()
End Sub
```
(There is no implements statement.)

Using Refactor rename (Foo -> Bar) results in the following.

Interface:
```vba
Public Sub Bar()
End Sub
```
Class:
```vba
Public Sub Interface_Bar()
End Sub
```

The rename in Class should not happen since we know that it does not implement Interface.

**Comment by BZngr** [comment]
This issue is fixed by #3111

**Comment by MDoerner** [comment]
Fix confirmed.
